server/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==75.3.0", "wheel", "Cython==3.0.12", "numpy==1.26.4"]
+requires = ["setuptools==75.3.0", "wheel", "Cython==3.0.12", "numpy==1.23.5"]
```

Comment:
I'd try hard to avoid this, since the latest wheels of medvedi build with 1.26.4, so we will lose the ablity to upgrade to PYthon 3.12.